mod_upload/controllers.py

```diff
@@ -222,6 +242,36 @@ def process_id(upload_id):
                     g.db.rollback()
                 # Move file
                 if db_committed:
+                    with open("issues.md", "r") as f:
+                        data = f.read()
+                    split_arr = data.split('**')
+                    content = split_arr[3] + split_arr[4] + \
+                        split_arr[5] + split_arr[6]
+                    if form.report.data == 'y':
+                        necessary_details = ('**{head}** \n '
+                                             'What platform did you use? '
+                                             '{platform} \n '
+                                             'What where the used arguments? '
+                                             '{arg} \n').format(
+                            head=split_arr[7], platform=form.platform.data,
+                            arg=form.parameters.data)
+                        sample_link = ('{server_id}/sample/'
```

Comment:
There are alternative (better) ways to obtain a full URL... Check how I'm sending out emails with the platform :)